rke/k8s_version_info.go

```diff
@@ -120,6 +120,12 @@ func loadK8sVersionInfo() map[string]v3.K8sVersionInfo {
 			MinRKEVersion:     "1.1.2-rc0",
 		},
 		// The Calico/Canal template in this version use functions that are only available in RKE v1.0.0 and up
+		// This version includes nodelocal dns only available in RKE v1.0.7 and up
+		"v1.16.11-rancher1-1": {
+			MinRancherVersion: "2.4.4-rc0",
```

Comment:
Update this to 2.4.5-rc0 as we are going to introduce @luthermonson's windows 1909 build support so these templates will only be applicable to 2.4.5+